internal/cluster/partition_service.go

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package cluster
+
+import (
+	"sync"
+	"sync/atomic"
+
+	"github.com/hazelcast/hazelcast-go-client/internal"
+	"github.com/hazelcast/hazelcast-go-client/internal/event"
+	ilogger "github.com/hazelcast/hazelcast-go-client/internal/logger"
+	"github.com/hazelcast/hazelcast-go-client/internal/proto"
+	iserialization "github.com/hazelcast/hazelcast-go-client/internal/serialization"
+	"github.com/hazelcast/hazelcast-go-client/internal/util/murmur"
+	pubserialization "github.com/hazelcast/hazelcast-go-client/serialization"
+)
+
+type PartitionServiceCreationBundle struct {
+	SerializationService *iserialization.Service
+	EventDispatcher      *event.DispatchService
+	startCh              chan struct{}
+	startChAtom          int32
+	Logger               ilogger.Logger
```

Comment:
These two fields startCh and StartChAtom looks unused. 
Is that intentional ?

Reply:
Removed.